app/javascript/packs/application.tsx

```diff
@@ -173,6 +173,10 @@ initReact({
       request={camelizeKeysAs<Request>(data.request)}
     />
   ),
+
+  'track-approaches': (data: any) => <TrackComponents.Approaches />,
+  'track-approach': (data: any) => <TrackComponents.Approach />,
```

Comment:
You could consider adding TODOs for things that someone else (me in this case) needs to do.